cmd/influx/main.go

```diff
@@ -1,12 +1,12 @@
 package main
 
 import (
-	"flag"
 	"fmt"
 	"os"
 
 	"github.com/influxdata/influxdb/client"
 	"github.com/influxdata/influxdb/cmd/influx/cli"
+	flag "github.com/spf13/pflag"
```

Comment:
This library is intended to be a drop-in replacement to the flag package and is supposed to be functionally identical. The user of the library isn't supposed to care if this is `pflag` or `flag`, which is why this is aliased. Because of the way this is meant to be used, it could also be easy to accidentally intermix two different packages without this alias.